drupalux/claro
# Add default block placing configuration to the theme

## Issue
Currently, when installing the theme the blocks aren't placed which means that the theme looks all messed up.

## Screenshot / UI changes
This change duplicates the Seven theme default block placing configuration for Claro.

## Testing instructions

Install the theme from scratch and see if the block placing configuration looks correct.
